examples/callback-results/Cargo.toml

```diff
@@ -8,13 +8,17 @@ edition = "2021"
 crate-type = ["cdylib"]
 
 [dependencies]
-near-sdk = { path = "../../near-sdk" }
+near-sdk = "4.1.1"
 
 [dev-dependencies]
-workspaces = "0.7.0"
+near-workspaces = { version = "0.8.0" }
 serde_json = "1.0"
 tokio = { version = "1.14", features = ["full"] }
 anyhow = "1.0"
+near-gas = { version = "*" }
```

Comment:
`Gas` is exposed through [`near_sdk::types::Gas`](https://github.com/near/near-sdk-rs/blob/e4c924058e9159ff54a5bfc4adaa8b79ba04d876/near-sdk/src/types/mod.rs#L13); do not import it separately.